README.md

```diff
@@ -16,6 +16,34 @@ If you would like waycorner to be available on your distro's package manager, fe
 paru -S waycorner
 ```
 
+
+### Nix/NixOS
+
+There are several methods to install waycorner using Nix. Waycorner already exists in Nixpkgs, so it is a matter of preference or
+available tooling. You may pick whichever option suites your use case.
+
+#### Nix (Standalone)
+
+```bash
+nix profile install nixpkgs#waycorner
+```
+
+#### NixOS
+
+```nix
+{pkgs, ...}: {
+    environment.systemPackages = [pkgs.waycorner];
+}
+```
+
+#### Home-Manager
+
+```nix
+{pkgs, ...}: {
+    home.packages = [pkgs.waycorner];
+}
+```
+
```

Comment:
```suggestion
### Nix(OS)

There is a `waycorner` package available via Nixpkgs. This is not automatically updated and is maintained by a kind individual unrelated to the project. Use with caution however.

```

1. I don't want to duplicate "how to use Nix" packaging here. People should imho know how to add it to their Nix setup.
2. Considering this is not including in the GitHub actions, this is not automatically updated so that is worth mentioning.
3. It's not maintained/owned by me, so it's worth calling that out too.

We can include it in the automatic updating if you could share some documentation on how that would be achieved via GitHub actions. Additionally, I'd need to be made owner of the Nix package.

Finally, can you move this under the "Cargo" installation?